install/battlesnake/install.yml

```diff
@@ -0,0 +1,18 @@
+id: battlesnake
+
+name: Custom Attributes
+
+title: |
+  APM: Report custom events and attributes
+
```

Comment:
Please add a summary as, so it's not blank on the catalog view.

Reply:
@jpvajda This is the install plan! So no summary needed here.